core/client/ui/toolboxes/level-toolbox.scss

```diff
@@ -10,4 +10,14 @@
     font-size: 0.85rem;
     margin: 15px 0;
   }
+
+  .feature-dropdown {
```

Comment:
Let's make it a bit larger:

![Screenshot from 2022-09-15 20-12-50](https://user-images.githubusercontent.com/835072/190499717-19b59fee-21de-41d5-8d24-6f8542b12cb6.png)